.github/workflows/update-configuration.yml

```diff
@@ -0,0 +1,58 @@
+name: "Update Configuration"
+
+on:
+  workflow_dispatch:
+  push:
+
+jobs:
+  update:
+    name: "Update Configuration in manifest.json"
+    runs-on: ubuntu-latest
+    permissions: write-all
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup node
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20.10.0"
+
+      - name: Install deps and run configuration update
+        run: |
+          yarn install --immutable --immutable-cache --check-cache
+          yarn tsc --noCheck --project tsconfig.json
+
+      - name: Update manifest configuration using GitHub Script
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const fs = require('fs');
+            const path = require('path');
+
+            const { pluginSettingsSchema } = require('./src/types/plugin-inputs');
+
+            const manifestPath = path.resolve("${{ github.workspace }}", './manifest.json');
+            const manifest = JSON.parse(fs.readFileSync(manifestPath, 'utf8'));
+
+            const configuration = JSON.stringify(pluginSettingsSchema);
+
+            manifest["configuration"] = JSON.parse(configuration);
+
+            const updatedManifest = JSON.stringify(manifest, null, 2)
+            console.log('Updated manifest:', updatedManifest);
+            fs.writeFileSync(manifestPath, updatedManifest);
+
+      - name: Commit and Push generated types
+        run: |
+          git config --global user.name 'ubiquity-os[bot]'
+          git config --global user.email 'ubiquity-os[bot]@users.noreply.github.com'
```

Comment:
The name associated to a commit can just be set that way yes. That is why on GitHub you can have the `Verified` badge next to a commit when that was done using the account auth, typically through GitHub website. Otherwise from simple git commit you can set any name.

Reply:
> Conclusion
> 
> You cannot commit to any repository as anyone’s GitHub App without authentication. The commit example you provided is authenticated via GitHub Actions’ GITHUB_TOKEN for that repository only, and you would need proper permissions and tokens to interact with any other repository.

I guess its possible because we added this app with write permissions to this repository. Good find! This seems super simple.